storybook/storybook-css/src/docs/focus.docs.mdx

```diff
@@ -0,0 +1,16 @@
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Docs/Designrichtlijnen/Focus staat" />
+
+# Focus staat
+
+De toegankelijkheidsrichtlijnen eisen dat het element op de pagina dat focus heeft visueel herkenbaar is. Dit geldt ook voor formuliervelden. Houd rekening bij het ontwerp en de ontwikkeling van de website of applicatie met de focus staat.
+
+Wij gebruiken grotendeels de standaard focus staat van de webbrowsers. De enige aanpassing die we doen is de focus `outline` een `offset` geven van 2 pixels, zodat er ruimte zit tussen het element en de focus staat. Hierdoor heeft de focus staat ook voldoende contrast met donkere elementen.
+
+## Relevante WCAG eisen
```

Comment:
More general as well: how can we manage the order of sections on a docs page? I’d say the example and API table should be almost at the top of the page. Instructions for usage, or guidelines to deal with can come below. No problem to fix this later.

Reply:
I agree, the general structure of docs pages can be improved. I'll add a ticket to our board. I think it's best to do this later though, when we have some more documentation pages. For now, our focus should be on making components, I think.